docs/assets/diagrams/linking/3-unlinking.puml

```diff
@@ -0,0 +1,58 @@
+@startuml Unlinking
+
+title: Unlinking
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+'start flow
+Alice -> App++: I want to unlink this bank account
+
+App -> Server ++: ""**DELETE /consents/123**""
+App <-- Server: ""202 Accepted""
+deactivate App
+
+Server -> Adapter ++: ""**DELETE /consents/123**""
+Adapter <-- Server: ""202 Accepted""
+deactivate Server
+
+Adapter -> Server++: ""**PATCH /consents/123** \n{\n status: string,\n revokedAt: string (of timestamp)\n}""
```

Comment:
can you update this to use `rnote` and real values? can refer to the PR #5

Reply:
yup sounds good
don't have any experience in PUML so bear with me haha

Reply:
okay, thank you! :)